e2e/test/scenarios/collections/collections.cy.spec.js

```diff
@@ -600,3 +608,11 @@ function moveItemToCollection(itemName, collectionName) {
     return collection.find(item => item.name === itemName);
   }
 }
+
+function createNestedCollection() {
```

Comment:
This creates another nested collection to move into `"First collection"`. I opted to do this procedure within the test, rather than adding it to the initial data set, because many other test rely on the exact setup we currently have (e.g. `[cy.findByText("6 items selected");`](https://github.com/metabase/metabase/blob/fix-move-undo/e2e/test/scenarios/collections/collections.cy.spec.js#L405)).

We could update the test data and the other test suites, but it would be a disproportionately large effort for this fix, and there may be tests in other files that break as well.